ignite/pkg/cosmosclient/cosmosclient.go

```diff
@@ -45,24 +48,33 @@ import (
 // is triggered prior to broadcasting but transfer's tx is not committed in the state yet.
 var FaucetTransferEnsureDuration = time.Second * 40
 
-var errCannotRetrieveFundsFromFaucet = errors.New("cannot retrieve funds from faucet")
+var (
+	// ErrInvalidBlockHeight is returned when a block height value is not valid.
+	ErrInvalidBlockHeight = errors.New("block height must be greater than 0")
+
+	errCannotRetrieveFundsFromFaucet = errors.New("cannot retrieve funds from faucet")
+)
 
 const (
 	defaultNodeAddress   = "http://localhost:26657"
 	defaultGasAdjustment = 1.0
 	defaultGasLimit      = 300000
-)
 
-const (
 	defaultFaucetAddress   = "http://localhost:4500"
 	defaultFaucetDenom     = "token"
 	defaultFaucetMinAmount = 100
+
+	defaultTXsPerPage = 30
+
+	searchHeight = "tx.height"
+
+	orderAsc = "asc"
 )
 
 // Client is a client to access your chain by querying and broadcasting transactions.
 type Client struct {
 	// RPC is Tendermint RPC.
-	RPC *rpchttp.HTTP
+	RPC rpcclient.Client
```

Comment:
```
// go:generate 
type RPCClient interface {
    rpcclient.Client
}
```

from our last call to gen mocks ourselves with the recent version of mockery so we can use `EXPECT()`